source/_integrations/thread.markdown

```diff
@@ -11,5 +11,12 @@ ha_integration_type: service
 ha_config_flow: true
 ---
 
-The Thread integration is currently a placeholder, but will manage Thread network credentials in a future release.
-The integration is automatically setup when the "Silicon Labs Multiprotocol" add-on is installed.
+The Thread integration helps you track the different Thread networks in your home, the credentials that they use and helps you align them. It is currently a work in progress.
+
+Thread is a low-power mesh networking standard which allows users to connect their devices within a home network. Thread allows devices to communicate with each other without the need for a central gateway or router. This makes it ideal for home automation, where a large number of devices may need to be connected.
+
+To connect a Thread network to Home Assistant, Thread Border Routers (TBRs) are used. TBRs are devices that bridge the Thread network to a local wi-fi or ethernet network. To add a TBR to Home Assistant, you can use our [Home Assistant Yellow](/yellow/) hub or the [Home Assistant SkyConnect](/skyconnect/) Zigbee/Thread stick.
```

Comment:
TBRs are devices that bridge the Thread network to a local Wi-Fi or Ethernet network.